setup.py

```diff
@@ -31,7 +32,7 @@
     name='hpack',
     version=version,
     description='Pure-Python HPACK header compression',
-    long_description=open('README.rst').read() + '\n\n' + open('HISTORY.rst').read(),
+    long_description=codecs.open('README.rst').read() + '\n\n' + codecs.open('HISTORY.rst',encoding='utf-8').read(),
```

Comment:
Oh, well README.rst is actually ascii-encoded, but opening it as UTF-8 won't hurt so OK.